F1/F1/CoreData/CoreDataManager.swift

```diff
@@ -0,0 +1,362 @@
+//
+//  CoreDataManager.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/05/15.
+//
+import CoreData
+import Foundation
+import UIKit
+
+class CoreDataManager {
+
+    // MARK: Variables
+    static let shared = CoreDataManager()
```

Comment:
singleons should be avoided. You can just create n object of CoreDataManager(0 and use DI to inject it into the repo